### ISSUE TITLE
[Avatar]: decorative property should be supported 

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents#9875]
### Is your feature request related to a problem?

Yes. In some cases Avatar is just used for decorative purpose and should be ignored by accessibility tools. 
For example: Avatar user name is shown in text beside the avatar, In this case Screen reader will read user name from text and also from Avatar accessibleName as shown below
![Screenshot 2024-09-13 at 4 42 33 PM](https://github.com/user-attachments/assets/598c6207-fbfa-4e4f-8f3b-326cc1fa5ae4)

### Describe the solution you'd like

Provide decorative property like : https://ui5.sap.com//#/api/sap.m.Avatar%23controlProperties

### Organization

SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @SAP/ui5-webcomponents-topic-p, take a look of this feature request.

Regards,
Iliana

**Comment by dobrinyonkov** [comment]
Hi @pankajpandey01,

Thanks for your feature request!
As per the new design requirements, this item should go through the inner source process. Could you please create a Jira item for Central Design and include the E1 reference number here?
Let me know if you have questions!

Thanks and Best Regards,
Dobrin

**Comment by pankajpandey01** [comment]
Hi @dobrinyonkov ,
I don't see any design change in this feature request. Its a a11y feature which will not affect design and it is already supported in UI5 https://ui5.sap.com//#/api/sap.m.Avatar%23controlProperties. So why create central design ticket is needed?

**Comment by dobrinyonkov** [comment]
Hi @pankajpandey01,

While the UI5 Avatar control supports this property, it is not currently part of the a11y specification. As such, the specification will need to be updated. Since the a11y specification is now managed by the design team, any updates or changes must go through their evaluation process.

Regards,
Dobrin

**Comment by pankajpandey01** [comment]
Hi @dobrinyonkov 
created central design ticket : FIORITECHE1-8671

Thanks,
Pankaj
